PetAdoption-iOS/PetAdoption-iOS/Extensions/UIImage+Placeholder.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIImage+Placeholder.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 8/20/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+import Alamofire
+import AlamofireImage
+
+extension UIImage
```

Comment:
No, it won't load them into an image view as that is the view controllers job. It simply returns the UIImage that has been created by the data downloaded.